cmd/standalone/main.go

```diff
@@ -130,6 +131,13 @@ func main() {
 
 			select {
 			case <-signalsChannel:
+				if closer, ok := spanWriter.(io.Closer); ok {
```

Comment:
what about spanReader, dependencyReader, etc. might be easier to register everything that implements the io.Closer interface during initialization and iterate over them and close them

Reply:
readers don't implement closer. This is just port from collector main.